src/test/librgw_file.cc

```diff
@@ -106,14 +106,11 @@ TEST(LibRGW, LOOKUP_BUCKETS) {
   if (! fs)
     return;
 
-  int ret = 0;
   for (auto& fid_row : bucket_matrix) {
     auto& fid = get<0>(fid_row);
     // auto& obj_vector = get<1>(fid_row);
     struct rgw_file_handle *rgw_fh = nullptr;
-    ret = rgw_lookup(fs, fs->root_fh, get<0>(fid).c_str(), &rgw_fh,
-		    0 /* flags */);
-    ASSERT_EQ(ret, 0);
+    ASSERT_EQ(0, rgw_lookup(fs, fs->root_fh, get<0>(fid).c_str(), &rgw_fh, 0 /* flags */));
```

Comment:
i agreed. we do not enforce this. but per https://github.com/ceph/ceph/blob/master/CodingStyle, we suggest using https://google.github.io/styleguide/cppguide.html as a baseline, which in turn puts this limit. see https://google.github.io/styleguide/cppguide.html#Line_Length